MyList/MyStringExtension.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Linq;
+
+
+namespace MyList
+{
+   public static class MyStringExtension
+   {
+      public static int[] ToArray(this string str, string enumerable)
```

Comment:
This method is expected to have one argument - you called it 'str' - and it should be converted.